aredis_om/model/model.py

```diff
@@ -1115,9 +1115,13 @@ def key(self):
         return self.make_primary_key(pk)
 
     @classmethod
-    async def delete(cls, pk: Any) -> int:
+    async def delete(cls, pk: Any, pipeline: Optional[Pipeline] = None) -> int:
```

Comment:
I think delete_all and delete should call a new function named ```_delete``` underneath, that accepts a kwargs (at least). This would be a bit cleaner.